### ISSUE TITLE
AirTrail on Kubernetes doesn't run Prisma migrations

### ISSUE THREAD

**Issue by krzwiatrzyk** [johanohly/AirTrail#71]
### The bug

![image](https://github.com/user-attachments/assets/4d013254-2224-4aa6-badb-dc02dd7ceca0)

AirTrail shows "Applying migrations" and database is still empty. Also, I have tried to run those migration myself, no ouput, hangs for few seconds, nothing happens.

### Operating System that is running AirTrail

Linux

### AirTrail Version

v0.3.1

### Your docker-compose.yml content

```YAML
None
```

### Your .env content

```Shell
ORIGIN + DB_URL
```

### Reproduction steps

1. Deploy AirTrail as a pod in Kubernetes

### Relevant log output

```shell
In the screenshot
```

### Additional information

It would be great to add some debug logs to migraitons

**Comment by krzwiatrzyk** [comment]
Not only an issue with Kubernetes, I guess it's related to an achitecture, it doesn't work with compose example on linux/amd64.

On linux/arm64 it works flawlessly.

**Comment by krzwiatrzyk** [comment]
I guess it's something wrong with `bunx prisma db` subcommand, it doesn't produce any output on `amd64`, If I run `npx prisma db push` it works correctly

**Comment by krzwiatrzyk** [comment]
Related I guess - https://github.com/oven-sh/bun/issues/5320

**Comment by krzwiatrzyk** [comment]
I can confirm that adding following line in Dockerfile:
```
COPY --from=node:20 /usr/local/bin/node /usr/local/bin/node
```

like this:
```
FROM base AS release
COPY --from=node:20 /usr/local/bin/node /usr/local/bin/node
COPY --from=install /temp/prod/node_modules node_modules
COPY --from=prerelease /app/build build
COPY --from=prerelease /app/package.json .
COPY --from=prerelease /app/prisma ./prisma

COPY docker-entrypoint.sh /app/bin/
RUN chmod +x /app/bin/docker-entrypoint.sh

USER bun
EXPOSE 3000/tcp
ENTRYPOINT [ "/app/bin/docker-entrypoint.sh" ]
```

fixes the issue

**Comment by Arcturuss** [comment]
Same issue when trying to install on amd64 host (Ubuntu on WSL) using default docker-compose file. 

relevant `prisma migrate` output when running it manually inside the container:

```
bun@43f8fd7c43df:/app$ export DEBUG="*"
bun@43f8fd7c43df:/app$ bunx prisma migrate deploy
prisma:engines binaries to download libquery-engine, schema-engine +201ms
prisma:get-platform Found distro info:
{
  "targetDistro": "debian",
  "familyDistro": "debian",
  "originalDistro": "debian"
} +2ms
prisma:get-platform Trying platform-specific paths for "debian" (and "ubuntu") +1ms
prisma:get-platform Found libssl.so file using platform-specific paths: libssl.so.1.1 +9ms
prisma:get-platform The parsed libssl version is: 1.1.x +0ms
bun@43f8fd7c43df:/app$ echo $?
0
```

Quick workaround until it's fixed: comment out `image` in docker-compose.yml, add `build` section, and create a Dockerfile.

docker-compose.yml:
```
    ...
    #image: johly/airtrail:latest
    build:
      dockerfile: Dockerfile
    ...
```

create Dockerfile:
```
from johly/airtrail:latest as airtrail-fixed
COPY --from=node:20 /usr/local/bin/node /usr/local/bin/node
```

**Comment by johanohly** [comment]
Thanks for the detailed report and for sharing the workaround! It’s great to see you found a solution. I’ll look into incorporating the fix for the node binary issue in the official image.

Regarding your workaround: could you confirm if it also works on amd64 systems? I don’t have any experience with arm builds, so your input here would help me validate the solution across both architectures. If the solution (workaround) is just to add that line to the dockerfile, that would obviously be very nice. 

Thanks again for your patience and contributions!

**Comment by johanohly** [comment]
This workaround has been released in v0.3.2

**Comment by johanohly** [comment]
> > Regarding your workaround: could you confirm if it also works on amd64 systems? I don’t have any experience with arm builds, so your input here would help me validate the solution across both architectures. If the solution (workaround) is just to add that line to the dockerfile, that would obviously be very nice.
> 
> Yes, workaround works in any platform

Thank you for confirming this yourself as well!
I ended up testing it myself and released the fix in v0.3.2. 

Thank you again for finding and debugging the bug.
